css/style.css

```diff
@@ -1,3 +1,84 @@
+/*general*/
+body {
+  font-family: Optima, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: linear-gradient(to bottom right, #d0c4ff, #846be6, #e6679c, #ffa5cb);
+  background-size: cover;
+  background-attachment: fixed;
+  /*background: no-repeat;*/
+}
 h1 {
-  color: purple;
+  color: #e6679c;
+  text-shadow: #84395d 1px 1px;
+}
+
+p {
+
+}
```

Comment:
Since you didn't end up using the p tag to define any CSS rules, you could either comment it out or delete it.  Not a big deal but figured I'd mention it.  It won't break anything as is though.